CodeQlFilters.yml

```diff
@@ -26,6 +25,7 @@
     "-MdeModulePkg/Universal/EbcDxe/EbcDebugger/EdbSupportString.c:cpp/uselesstest",
     "-MdeModulePkg/Universal/EbcDxe/EbcDebugger/EdbSupportString.c:cpp/uselesstest",
     "-MdeModulePkg/Universal/RegularExpressionDxe/oniguruma/src/**/*.c:*",
+    "-MdeModulePkg/Universal/Console/ConPlatformDxe/ConPlatform.c:SM02311",

```

Comment:
Can you please keep the list lexicographically sorted?

Reply:
Sorted